### PR TITLE
feat: treasury page

### PR DESCRIPTION
**Pull request by evgenibir**
### 🗃 Github Issue Or Explanation for this PR. (What is it supposed to do and Why is needed)

https://linear.app/hypha/issue/DEV-654/add-api-integration
https://linear.app/hypha/issue/DEV-626/add-treasury-multi-sig-table
https://linear.app/hypha/issue/DEV-665/css-for-treasury-multi-sig-table

### ✅ Checklist

- Added UI with backend integration for treasury page

### 🙈 Screenshots
image
image
image
image

closed DEV-654
closed DEV-626
closed DEV-665
fixed DEV-700
fixed DEV-701

### CODE REVIEW

File: src/pages/dho/Treasury.vue [hunk 1]
```diff
@@ -68,10 +90,35 @@ export default {
       sort: 'Sort by last added',
       circle: 'All circles',
       optionArray: ['Sort by last added'],
-      circleArray: ['All circles', 'Circle One']
+      circleArray: ['All circles', 'Circle One'],
+      tab: 'PAYOUT',
```
Comment (Alex-IS) [inline src/pages/dho/Treasury.vue:94]: @evgenibir Constant instead of string

File: src/store/dao/actions.js [hunk 2]
```diff
@@ -267,6 +268,101 @@ export const updateDAOSettings = async function (context, { docId, data, alerts,
...
+        msig_id: data?.[0].msigId
+      }
+    }
+  ]
+  return this.$api.signTransaction(actions)
+}
+export const executeMultisigPay = function (context, { data }) {
+  const actions = [
+    {
+      account: 'eosio.msig',
```
Comment (Alex-IS) [inline src/store/dao/actions.js:351]: @evgenibir do not hardcode account name, provide it as environment variable

**Review by Alex-IS — COMMENTED** [review]
Just some standard comments :)

File: src/pages/dho/Treasury.vue (hunk 1 above)
Comment (evgenibir) [reply src/pages/dho/Treasury.vue]: @Alex-IS fixed

File: src/pages/dho/Treasury.vue [hunk 3]
```diff
@@ -298,6 +402,56 @@ export default {
...
       this.pagination = pagination
+    },
+    async createMultisig () {
+      try {
+        await this.createMultisigPay({ treasuryId: this.treasuryId, payments: this.selected, treasuryAccount: this.treasuryAccount })
+        this.successfullMultisigTransaction = true
+        this.selected = []
+        await this.$apollo.queries.redemptions.refetch()
+        await this.$apollo.queries.daoMultisigSignRequestsQuery.refetch()
+        this.tab = 'MULTISIG'
```
[the remark this answers is not in the document]
Comment (evgenibir) [reply src/pages/dho/Treasury.vue]: @Alex-IS fixed

File: src/store/dao/actions.js (hunk 2 above)
Comment (evgenibir) [reply src/store/dao/actions.js]: @Alex-IS fixed

File: src/store/dao/actions.js [hunk 4]
```diff
@@ -267,6 +268,101 @@ export const updateDAOSettings = async function (context, { docId, data, alerts,
...
+      }
+    }
+  ]
+  return this.$api.signTransaction(actions)
+}
+export const approveMultisigPay = function (context, { data }) {
+  console.log(data[0].msigId)
+  const actions = [
+    {
+      account: 'eosio.msig',
```
[the remark this answers is not in the document]
Comment (evgenibir) [reply src/store/dao/actions.js]: @Alex-IS fixed

**Pull request merged**